src/data_management/rasters/Raster.js

```diff
@@ -184,71 +185,77 @@ class Raster extends Component {
-        this.getRastersFromApi(
-          this.state.page,
-          this.state.searchTerms,
-          this.state.include3diScenarios
-        );
+      const checkedUuids = this.state.checkboxes.filter(checkbox => checkbox.checked).map(checkbox=>checkbox.raster.uuid);
+      const opts = {
+        // not permanently deleted, this will be implemented in backend
+        credentials: "same-origin",
+        method: "DELETE",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({})
+      };
+      this.fetchRasterUuidsWithOptions(checkedUuids, opts);
+    }
+  }
 
-        // TODO show user results of the delete
-        // the addNotification seems not to support translation and also may not be suitable for long text
-        // should we use modal instead ?
-        // const valuesWithRasters = values.map((el, ind)=> {return{
-        //   promise: el,
-        //   rasterUuid: this.state.rasters.find(e=> e.uuid === toBeDeletedRasterUuidsArray[ind])
-        // }});
-        // const failedDeletes = valuesWithRasters.filter(e => e.promise.ok === false);
-        // const succeededDeletes = valuesWithRasters.filter(e => e.promise.ok === true);
-        // // reduce to userpresentable string
-        // let deleteNotificationStr = '';
-        // if (succeededDeletes.length > 0) {
-        //   deleteNotificationStr += 'following rasters deleted'
-        // }
-        // // dispatch notification
-        // // is this notification evn working anywhere?
-        // this.props.addNotification("rasters deleted", 2000);
-      });
+  handleFlushDataRasterClick() {
+    if (
+      window.confirm(
+        "Are you sure you want to remove all data in the next raster(s)? \n  \n " +
+        this.state.checkboxes.filter(checkbox => checkbox.checked).map(checkbox=>checkbox.raster.name).join(" \n")
+      )
+    ) {
+      const checkedUuids = this.state.checkboxes.filter(checkbox => checkbox.checked).map(checkbox=>checkbox.raster.uuid);
+      const opts = {
+        // not permanently deleted, this will be implemented in backend
+        credentials: "same-origin",
+        method: "PATCH",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify({"source": null} )
+      };
+      this.fetchRasterUuidsWithOptions(checkedUuids, opts);
     }
   }
 
+  fetchRasterUuidsWithOptions(uuids, fetchOptions) {
+    const url = "/api/v4/rasters/";
+    // array to store all fetches to later resolve all promises
+    const fetches = uuids.map (rasterUuid => {
+      return (fetch(url + rasterUuid + "/", fetchOptions));
+    });
+    Promise.all(fetches).then(values => {
+      // Refresh the page, so that the removed rasters are no longer visible
+      this.getRastersFromApi(
+        this.state.page,
+        this.state.searchTerms,
+        this.state.include3diScenarios
+      );
+    });
+    // TODO show user results of the fetch
```

Comment:
I see a lot of comments here. I suppose these comments are to implement a new feature of the data management and maybe you want to keep them here to do it later on?

Reply:
Yep I think it would be nice to give the user some specific feedback about which rasters are deleted. 
It was however a bit too complicated at first.
But maybe we could have a simpler message just saying: "rasters deleted" or something